src/crimson/os/seastore/transaction_manager.h

```diff
@@ -335,6 +335,74 @@ class TransactionManager : public ExtentCallbackInterface {
     });
   }
 
+  /**
+   * remap_pin
+   *
+   * Remap original extent to new extent.
+   * Return the pin of new extent.
+   */
+  using remap_pin_iertr = base_iertr;
+  using remap_pin_ret = remap_pin_iertr::future<LBAPinRef>;
+  template <typename T>
+  remap_pin_ret remap_pin(
+    Transaction &t,
+    LBAPinRef &&pin,
+    extent_len_t new_offset,
+    extent_len_t new_len) {
```

Comment:
The names of new_offset/new_len are confusing in remap_pin() and overwrite_pin() because they have different meaning.

Probably change to remap_offset/remap_len in remap_pin(), and overwrite_offset/overwrite_len in overwrite_pin().